locale/circuitpython.pot

```diff
@@ -1931,6 +1935,10 @@ msgstr ""
 msgid "Pin is input only"
 msgstr ""
 
+#: ports/atmel-samd/common-hal/alarm/pin/PinAlarm.c
+msgid "Pin is not free"
```

Comment:
Is there a `Pin in use` message you can use instead? Adding another message takes more space and requires new translations.